### ISSUE TITLE
Remove all items in selector mode

### ISSUE THREAD

**Issue by acanthis** [TerryZ/v-selectpage#68]
Hello! 
How can remove all values using "selector" mode? Why is there no "remove" method in this mode?

**Comment by TerryZ** [comment]
Just directly set v-model value to empty array to remove all selected items

template

```vue
<template>
  <SelectPageList v-model="selected" />
  <button type="button" @click="removeAll">remove all</button>
</template>
```

script

```js
import { ref } from 'vue'

const selected = ref([])

function removeAll () {
  selected.value = []
}
```

**Comment by acanthis** [comment]
Unfortunately, it don't work for me. The values from the component are not deleted.

https://stackblitz.com/edit/vue-l4lafc?file=src%2Fcomponents%2FHelloWorld.vue

**Comment by TerryZ** [comment]
Fixed in `v3.0.1`
